### PR TITLE
feat: Use deeplink for the web version of the bar

### PR DESCRIPTION
**Pull request by Crash--**
The idea is to open from the "mobile web app" the native app if installed. If not, we redirect to the web version.

### CODE REVIEW

**Comment by Crash--** [comment]
[(𝐒) Cozy-bar & deeplink](https://trello.com/c/zQIRE3tI/1686-%F0%9D%90%92-cozy-bar-deeplink)

File: src/components/Apps/AppItem.jsx [hunk 1]
```diff
@@ -99,6 +101,17 @@ export class AppItem extends React.Component {
       // target app is web
       // run switch listener only if the current app is native mobile
       onClick = this.onAppSwitch
+    } else if (isMobile()) {
```
Comment (CPatchane) [inline src/components/Apps/AppItem.jsx]: What is the difference with the `isMobileApp()` call?

Comment (Crash--) [reply src/components/Apps/AppItem.jsx]: `isMobileApp` checks the cordova context aka the native one. 

Here, we just test if it's the mobile web version... Thanks cozy-device-helper and its name :/

Comment (CPatchane) [reply src/components/Apps/AppItem.jsx]: It should be more explicit on cozy-device-helper, but I won't block this PR for that

Comment (y-lohse) [reply src/components/Apps/AppItem.jsx]: But hang on, `isMobileAppAvailable ` can only be checked when `isMobile()` is `true`, right? So maybe we could make this triple condition a bit clearer?

Comment (Crash--) [reply src/components/Apps/AppItem.jsx]: In fact the three tests are: 
- if we are on native app and other native app is installed
- else if we are on native app 
- else if we are on mobile 

How can I make them clearer? By renaming the variables?

Comment (y-lohse) [reply src/components/Apps/AppItem.jsx]: Yeah I think renaming the variables or being more explicit in the commend would help. I know it's not part of this PR but `run switch listener` for example is not very clear.

Something like this I think would help:

```
const isNativeToNative = isMobileAppAvailable
const isNativeToWeb = isMobileApp()
const isWebToNative = isMobile()
```

File: src/components/Apps/AppItem.jsx [hunk 2]
```diff
@@ -99,6 +101,17 @@ export class AppItem extends React.Component {
...
+      //If we are on the "mobile web version", we try to open the native app
+      //if it fails, we redirect to the web version of the requested app
+      onClick = e => {
+        if (NATIVE_APP_INFOS[app.slug]) {
```
Comment (y-lohse) [inline src/components/Apps/AppItem.jsx]: Nit: this condition might be better with the `if (isMobile())`.

**Pull request merged**